### DISCUSSION TITLE
Difficulty sorting doesn't seem to work 🤔

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#19319] [Q&A] [answered]
I noticed that I have a lot of beatmaps with -1.00 difficulty
After clicking in, all difficulty are not sorted
I'm playing on Windows 11
![屏幕截图 2022-07-23 172216](https://user-images.githubusercontent.com/74862110/180599350-07b080af-b5ee-4680-8d42-93114f49fae3.png)

**Comment by frenzibyte** [comment, marked as answer]
Please attach logs.

**Reply by Jan-Arbus** [reply]
[network.log](https://github.com/ppy/osu/files/9173479/network.log)
[runtime.log](https://github.com/ppy/osu/files/9173480/runtime.log)
[database.log](https://github.com/ppy/osu/files/9173481/database.log)
[performance.log](https://github.com/ppy/osu/files/9173482/performance.log)
[updater.log](https://github.com/ppy/osu/files/9173483/updater.log)

**Reply by peppy** [reply]
Looks like background processing died with an exception, but the exception isn't being logged.

**Reply by cdwcgt** [reply]
One of the reasons for his breakdown is database have a beatmap which use not available ruleset
part of my log:
```
Background processing failed on unknown artist - unknown title (cdwcgt): osu.Game.Rulesets.RulesetLoadException: Ruleset could not be loaded (Ruleset not available) 
```
because not catch, an entire task stops running (fixed after https://github.com/ppy/osu/commit/f1791e79e3e6cb607867304a1b560937f5876ba2)

after catching exception,"foreach" statement can continue to run.
So the problem should be solved after all beatmap is fully processed

**Reply by frenzibyte** [reply]
Tracking anything left in #19325, although I think it should all be resolved now.
